src/pilot/templates/nic-configs/4_port/controller.yaml

```diff
@@ -104,6 +104,39 @@ parameters:
     default: mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
     description: The bonding_options string for the bond interface.
     type: string
+  InternalApiMTU: # Override this via parameter_defaults
+    default: 1500
+    description: MTU value for this network.
+    type: number
+  StorageNetworkMTU: # Override this via parameter_defaults
+    default: 1500
+    description: MTU value for this network.
+    type: number
+  StorageMgmtNetworkMTU: # Override this via parameter_defaults
+    default: 1500
+    description: MTU value for this network.
+    type: number
+  TenantNetworkMTU: # Override this via parameter_defaults
+    default: 1500
+    description: MTU value for this network.
+    type: number
+  ProvisioningNetworkMTU: # Override this via parameter_defaults
+    default: 1500
+    description: MTU value for this network.
+    type: number
+  ManagementNetworkMTU: # Override this via parameter_defaults
+    default: 1500
+    description: MTU value for this network.
+    type: number
+  DefaultBondMTU: # Override this via parameter_defaults
+    default: 9216
```

Comment:
9000?

Reply:
On switch ports, MTU size is set to 9216. Also, all bonds on different nodes carry different VLANs. And bonds MTU should be greater than the VLANs MTU to allow overhead of different encapsulations. That is why we set it to MAX possible MTU on the switches.